Packs/Slack/ReleaseNotes/3_5_15.md

```diff
@@ -0,0 +1,6 @@
+#### Scripts
+
+##### SlackAskV2
+
+- Documentation and metadata improvements.
```

Comment:
Maybe let's specify here also what we added to the docs - how to use the SlackAskv2 with a playbook. To be more concise